src/klicky.ts

```diff
@@ -1,3 +1,17 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+let count = 0;
+export const click = () => {
+  console.log("You Clicked!");
+  count+=1;
+  document.getElementById("counter").innerHTML = "Wow! You Clicked " + count + " Time(s)";
+  document.getElementById("counter").style.fontFamily = "Courier New";
```

Comment:
haha this is cute for the index.html page, but isn't it a pain to add all these styles using JS? Try looking at how you can do this using CSS without any JS by targeting the `counter` ID